### DISCUSSION TITLE
Add ability to pass `select` argument to Prisma Client in adapter-prisma API

### DISCUSSION THREAD

**Discussion by MurzNN** [keystonejs/keystone#5306] [Feature requests]
Prisma Client have ability to limit list of returning fields via [select](https://www.prisma.io/docs/concepts/components/prisma-client/select-fields) API argument. This is useful for decrease the transferred data size and increase SQL query execution.

But Keystone's adapter-prisma API at now don't provide ability to pass the `select` argument to Prisma Client. Will be good to add this feature.

**Comment by bladey** [comment]
Thank you for the feature suggestion @MurzNN.
